README.md

```diff
@@ -24,8 +24,7 @@ The table below shows a list of PostgresKit major releases alongside their compa
 
 |Version|NIO|Swift|SPM|
 |-|-|-|-|
-|2.0|2.0|5.2+|`from: "2.0.0"`|
-|1.0|1.0|4.0+|`from: "1.0.0"`|
+|2.0|2.0|5.8+|`from: "2.0.0"`|
```

Comment:
isn't it incorrect to specify "5.8+" for "2.0.0"? Should just be 5.2+.

Reply:
This PR bumps the minimum requirement for PostgresKit to 5.8, so this is correct as is.

Reply:
2.0.0 still supports 5.8 though. You'll need to mention the correct number were requiring 5.8 started at. Like 2.14.

Reply:
2.0 supported 5.1, so this was never correctly done from day one and I don't care enough to bother now any more than anyone other than you does 🙂

Reply:
sorry that first swift version was supposed to be 5.2 (although you said it's 5.1)

Reply:
There, happy now? :trollface: